libs/editor/WordPressEditor/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+
+    <!-- Gutenberg -->
+    <color name="background_color">#121212</color>
```

Comment:
While overriding color values is a legit approach, ideally we would use attributes for this, that we could override in `styles.xml` on client-side. If GB would use other standard material color attributes (`colorPrimary`, `colorSecondary`,`colorBackground`, etc.), styling it from the client would also be a breeze.